### DISCUSSION TITLE
Full featured floating menu

### DISCUSSION THREAD

**Discussion by fredmabire** [sibiraj-s/ngx-editor#388] [Q&A] [answered]
Hi @sibiraj-s 

Is it possible to make bubble component to have full menu features or at least background color and font color ?

Can I implement this ?

Thanks

**Comment by sibiraj-s** [comment, marked as answer]
Hi,

You can literally render anything inside floating menu. So to have a full featured menu. you can render `ngx-editor-menu` inside `ngx-floating-menu`.

See here for example. https://stackblitz.com/edit/ngx-editor-floating-menu-custom

Hope this helps...

**Reply by fredmabire** [reply]
Really help ! Thanks !

[a comment by an automated account is not shown]
